CHANGES.md

```diff
@@ -1,3 +1,23 @@
+## edge-19.1.4
+
+* Controller
+  * Added timeouts support! Configurable in the service profiles for each route
+  * Added an experimental CNI plugin to avoid requiring the NET_ADMIN capability
+    when injecting proxies (more details at https://linkerd.io/2/cni)
```

Comment:
We should give `@codeman9` a shoutout here for all of his hard work.

Reply:
will do